command-line-interface.md

```diff
@@ -0,0 +1,246 @@
+# Operations
+
+The [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Tue, Oct 25, 2016 at 11:41:36AM -0700, Mike Brown wrote:
> As with the above description, I think it's important to say something like "for the protocols/apis it implements" as a qualification.

I think we [have that now][1] with the “[at least one API]” [1] and “APIs defined by this specification are” [2] language.  If you feel that langauge could be improved, can you suggest new language?

> Then at the title of the cli-api spec say something like the requirements specified in this section/document/specification are for implementations of the cli-api spec.

I've taken a stab at this with e276634 → 5c981a8.  If/when #570 lands we can make the README link more specific.

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#diff-04c6e90faac2675aa89e2176d2eec7d8R36
[2]: https://github.com/opencontainers/runtime-spec/pull/513/files#diff-04c6e90faac2675aa89e2176d2eec7d8R40